core/templates/domain/topic/editable-topic-backend-api.service.spec.ts

```diff
@@ -189,7 +188,7 @@ describe('Editable topic backend API service', () => {
     fakeAsync(() => {
       let successHandler = jasmine.createSpy('success');
       let failHandler = jasmine.createSpy('fail');
-      let topic = null;
+      let topic!: TopicBackendDict;
```

Comment:
Please add a comment why the `!` is needed.

Reply:
Done!